### ISSUE TITLE
Wrong SlideCurrentNo on slides export

### ISSUE THREAD

**Issue by Chadys** [slidevjs/slidev#1935]
**Describe the bug**

When exporting a presentation to pdf, the component `SlideCurrentNo` print "1" for every slides

**Minimal reproduction**

1. have a presentation with several pages
2. Add the following to a file `global-top.vue`
```html
<template>
  <footer class="absolute bottom-0 right-0 m-4"><small><SlideCurrentNo/>/<SlidesTotal/></small></footer>
</template>
```
4. Export using command `slidev export`

Every slides will have 1/X written in their bottom right corner instead of the correct current page number

**Environment**

- Slidev version: v0.50.0-beta.5 
- Browser: not pertinent, export issue
- OS: MacOS Sonova 14.5

**Comment by KermanX** [comment]
`global-top` and `global-bottom` are special because they're supposed to be singleton. There are two ways to solve this:
- Adding a `--per-slide` option when exporting
- Use `slide-top` instead of `global-top` (https://sli.dev/features/global-layers)

**Comment by Chadys** [comment]
Oh you're right, I used the `--per-slide` option and it worked perfectly for the slide number.
It would be good for this option to be documented in the [exporting guide](https://sli.dev/guide/exporting), I missed the warning about it in [global layers doc](https://sli.dev/features/global-layers) when I was setting it up because I had no idea then I would need the ability to export.
Once I needed to export, I only read the exporting guide and all its option there.
